pkg/generated/bindata.go

```diff
@@ -13651,7 +13651,7 @@ msgstr "Imposta il current-context in un file kubeconfig"
 
 #: pkg/kubectl/cmd/describe.go:86
 msgid "Show details of a specific resource or group of resources"
-msgstr "Mostra i dettagli di una specifiche risorsa o un gruppo di risorse"
+msgstr "Mostra i dettagli di una specifica risorsa o un gruppo di risorse"
```

Comment:
unexpected change. Make it another commit at least if this fix is right.

Reply:
This was not intended. I think it was caused by running update-codegen.sh against the current master.

Reply:
Do we have a spell checker included now? :)

Reply:
Looks like it https://github.com/kubernetes/kubernetes/blob/3ea4e449c361a66eb3539433487cd22d7256b032/hack/verify-spelling.sh
:-)

Reply:
Wow. Anyway, saw the same issue. The checked in bindata is broken it seems.

Reply:
Separated it in another commit.